.github/workflows/docs-update.yml

```diff
@@ -0,0 +1,36 @@
+name: Update OpenTelemetry Website Docs
+
+on:
+  # triggers only on a manual dispatch
+  workflow_dispatch:
+
+jobs:
+  update-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: make-pr
+        env:
+          API_TOKEN_GITHUB: ${{secrets.GITHUB_PAT_TOKEN}}
+          # Destination repo should always be 'open-telemetry/opentelemetry.io'
+          DESTINATION_REPO: open-telemetry/opentelemetry.io
```

Comment:
I don't think any of the maintainers here will be able to push a branch to this repo - it'd probably need to itself be a personal fork. I think it's a bit too complicated, let me comment on the issue to see if we can improve it.

Reply:
Maybe it should be my PAT?